### PR TITLE
Add info to readme file for correct applying changes to the own theia-image

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
* Add necessary steps for correct applying changes to own Che-Theia image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ CI for PR job in this repository will use `build.sh --pr`.
 If you want to publish docker images use `build.sh --push`
 
 ### How to build own che-theia image with Docker
+Make sure that you have added name of your branch into che-theia-init-sources.yml (`checkoutTo` section).
```
Comment (l0rd) [inline README.md]: Not a big deal but a best practice in markdown is to [surround headers with blank lines]( https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines)

Comment (musienko-maxim) [reply README.md]: done

**Comment by musienko-maxim** [comment]
check

**Comment by musienko-maxim** [comment]
not actual

**Pull request closed without merging**